t5x/configs/runs/finetune_mnli.gin

```diff
@@ -0,0 +1,151 @@
+# Defaults for finetuning with train.py.
```

Comment:
These 3 gin files in the configs/runs should be moved. This directory should contain the most generic run files